src/Umbraco.Web/Search/IInternalSearchConstants.cs

```diff
@@ -0,0 +1,13 @@
+using System.Collections.Generic;
+
+namespace Umbraco.Web.Search
+{
+    public interface IInternalSearchConstants
+    {
+        List<string> GetBackOfficeFields();
```

Comment:
These all need to be immutable (i.e. IEnumerable<string> or IReadOnlyList<string> or IReadOnlyCollection<string>)

Reply:
Sure I will change them :) to IEnumerable

Reply:
Changed @Shazwazza